.githooks/pre-commit

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+make lint
```

Comment:
missing new line

Reply:
[Ironic really, he could save others from coding errors but not himself](https://i.ytimg.com/vi/ApPs5ZWP_Ac/maxresdefault.jpg)